src/components/nodes/startEvent/shape.js

```diff
@@ -11,14 +12,26 @@ export default shapes.standard.Circle.extend({
   }, {
     tagName: 'image',
     selector: 'image',
-  }],
+  },
+  markersMarkup('topLeft'),
+  markersMarkup('topCenter'),
+  markersMarkup('topRight'),
+  markersMarkup('bottomLeft'),
+  markersMarkup('bottomCenter'),
+  markersMarkup('bottomRight')],
 
   defaults: util.deepSupplement({
 
     type: 'processmaker.components.nodes.startEvent.Shape',
     size: { width: 36, height: 36 },
     attrs: {
       'image': { 'ref-x': 5, 'ref-y': 5, ref: 'circle', width: 26, height: 26 },
+      ...markersAttrs('topLeft', { ref: 'circle', 'ref-y': -20, 'ref-padding-x': 0 }),
+      ...markersAttrs('topCenter', { ref: 'circle', 'ref-y': -20 }),
+      ...markersAttrs('topRight', { ref: 'circle', 'ref-x': 26, 'ref-y': -20, 'ref-padding-x': 0 }, -1),
+      ...markersAttrs('bottomLeft', { ref: 'circle', 'ref-padding-x': 0, 'ref-padding-y': -40 }),
+      ...markersAttrs('bottomCenter', { ref: 'circle', 'ref-padding-y': -40 }),
+      ...markersAttrs('bottomRight', { ref: 'circle', 'ref-x': 26, 'ref-padding-x': 0, 'ref-padding-y': -40 }, -1),
     },
 
   }, shapes.standard.Circle.prototype.defaults),
```

Comment:
You make a call to `this.refreshMarkers` for _Task_ and _Call Activity_, but not for _Start Event_, why is that?